### DISCUSSION TITLE
could you please add and 'opacity' setting i would love to have to app open as an overlay while gaming !

### DISCUSSION THREAD

**Discussion by Pullicid** [seerge/g-helper#1421] [Ideas]
love the software this my tuf dash f15 would be useless without it

**Comment by seerge** [comment]
@Pullicid hello, I honestly don't think that G-Helper is a good "monitoring" tool, there is a lot of very useful overlays for games (like afterburner+rivatuner , amd or nvidia overlays, etc)

But if you want to make window transparent you can try some apps from this list https://www.makeuseof.com/tag/peek-through-make-any-window-transparent-using-a-keyboard-shortcut/
